server/src/views/Login.vue

```diff
@@ -49,8 +63,27 @@ import Navbar from '@/components/Navbar.vue'; // @ is an alias to /src
           link: '/signup',
         },
       ],
+      loginData: {
+        username: '',
+        password: '',
+      },
     };
   },
+  methods: {
+    attemptLogin( event ) {
+      console.log( event );
+      this.$http.post('/login', {
+        username: this.$data.loginData.username,
+        password: this.$data.loginData.password,
+      }).then((res) => {
+        console.log('help me');
+        console.log(res);
```

Comment:
Ideally we would route to the main page here, but this is fine for now.

Reply:
Also, why does this seem so easy when I'm not writing it? 😂

Reply:
We're we getting a 200 OK from that request? I guess only when Flask is running. But yeah, it'll redirect.

Reply:
Honestly idk, maybe we were burnt out on Sun. Even the TS (shocked face) seems to be getting easier to read and write.